manifests/a/Airytec/SwitchOff/3.5.1.950/Airytec.SwitchOff.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Airytec.SwitchOff
+PackageVersion: 3.5.1.950
+PackageLocale: en-US
+Publisher: Airytec
+PackageName: Switch Off
+License: N/A

```

Invalid License type
```suggestion
License: Proprietary
```